packages/fiori-freestyle-writer/test/__snapshots__/basic.test.ts.snap

```diff
@@ -3287,7 +3286,7 @@ export function createDeviceModel () {
     <!-- Bootstrap the UI5 core library. 'data-sap-ui-frameOptions=\\"allow\\"'' is a NON-SECURE setting for test environments -->
     <script id=\\"sap-ui-bootstrap\\"
         src=\\"../resources/sap-ui-core.js\\"
-        data-sap-ui-libs=\\"sap.m,sap.ui.core,sap.ushell\\"
+        data-sap-ui-libs=\\"sap.m,sap.ui.core\\"
```

Comment:
Need to check into this. It doesn't look right to me. flpSandbox.html should always have `sap.ushell`

Reply:
Handled with suggested code - https://github.com/SAP/open-ux-tools/pull/2280/commits/97603f60d27230aef1529859db018a9441a332b2

Thanks